BUILD_INSTRUCTIONS.md

```diff
@@ -31,10 +31,19 @@ Before building the driver, the build files for the system must be generated, th
 
 As project uses AWS services for AWS SIGV4 authentication, you must [provide AWS credentials](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/credentials.html). 
 
+### Setting up AWS SDK 
+
+Setup on Windows 
```

Comment:
Change these to level-4 headers and simplify
```suggestion
#### Windows 
```

Reply:
done